crates/router/src/connector/utils.rs

```diff
@@ -272,7 +272,8 @@ impl PaymentsAuthorizeRequestData for types::PaymentsAuthorizeData {
     fn is_auto_capture(&self) -> Result<bool, Error> {
         match self.capture_method {
             Some(diesel_models::enums::CaptureMethod::Automatic) | None => Ok(true),
-            Some(diesel_models::enums::CaptureMethod::Manual) => Ok(false),
```

Comment:
This is being used by multiple connectors, making changes here will impact all those connectors